src/session.cpp

```diff
@@ -328,7 +335,7 @@ session_t::memory_pressure() const {
 
 std::string
 session_t::name() const {
```

Comment:
We can return optional here, instead of strange strings. What if we have service named "<none>"?